tests/arp/conftest.py

```diff
@@ -139,10 +139,15 @@ def intfs_for_test(duthosts, enum_rand_one_per_hwsku_frontend_hostname, enum_fro
 
 
 @pytest.fixture(scope="module")
-def common_setup_teardown(duthosts, ptfhost, enum_rand_one_per_hwsku_frontend_hostname):
+def common_setup_teardown(duthosts, ptfhost, enum_rand_one_per_hwsku_frontend_hostname, enum_frontend_asic_index, tbinfo):
     try:
         duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
-        router_mac = duthost.shell('sonic-cfggen -d -v \'DEVICE_METADATA.localhost.mac\'')["stdout_lines"][0].decode("utf-8")
+        if tbinfo["topo"]["type"] == "t2":
+            config_facts = duthost.asic_instance(enum_frontend_asic_index).config_facts(host=duthost.hostname, source="running")['ansible_facts']
```

Comment:
we can remove topo check . should work for any topo

Reply:
Hi @abdosi, Made the change as you suggested.